### PR TITLE
Exampleios 76

### PR DESCRIPTION
**Pull request by kfa-onewelcome**

### CODE REVIEW

File: Configuration/SecurityController.m [hunk 1]
```diff
@@ -7,7 +7,4 @@ +(bool)rootDetection{
 +(bool)debugDetection{
   return NO;
 }
-+(bool)debugLogs{
-  return YES;
```
Comment (szymon-onewelcome) [inline Configuration/SecurityController.m:11]: Isn't this filem modified by the configuration script? Was this method removed by it?

File: OneginiExampleAppSwift/Assembly/PresenterAssembly.swift [hunk 2]
```diff
@@ -92,5 +93,12 @@ class PresenterAssembly: Assembly {
 
         container.register(UINavigationController.self) { _ in UINavigationController() }.inObjectScope(.container)
         container.register(TabBarController.self) { _ in TabBarController() }.inObjectScope(.container)
+        container.register(UIWindow.self) { _ in
+            let window = UIWindow(frame: UIScreen.main.bounds)
```
Comment (szymon-onewelcome) [inline OneginiExampleAppSwift/Assembly/PresenterAssembly.swift:97]: Not `LazyWindow`?

File: OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift [hunk 3]
```diff
@@ -44,7 +44,7 @@ extension RegisterUserInteractor: RegisterUserInteractorProtocol {
     }
 
     func startUserRegistration(identityProvider: ONGIdentityProvider? = nil) {
-        ONGUserClient.sharedInstance().registerUser(with: identityProvider, scopes: ["read", "write"], delegate: self)
+        ONGUserClient.sharedInstance().registerUser(with: identityProvider, scopes: ["read", "openid"], delegate: self)
```
Comment (szymon-onewelcome) [inline OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: Can we have an enum with all possible options?

File: widget/ResourceGateway.swift [hunk 4]
```diff
@@ -27,7 +27,8 @@ class ResourceGateway {
         let implicitRequest = ONGResourceRequest(path: "user-id-decorated", method: "GET")
         ONGUserClient.sharedInstance().fetchImplicitResource(implicitRequest) { response, error in
             guard let data = response?.data,
-                  let responseData = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String] else {
+                  let responseJsonData = try? JSONSerialization.jsonObject(with: data, options: []) as? [String: String],
+                  let responseData = responseJsonData else {
```
Comment (szymon-onewelcome) [inline widget/ResourceGateway.swift:31]: What is the point of this?

File: OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift (hunk 3 above)
Comment (szymon-onewelcome) [inline OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: .

**Review by szymon-onewelcome — CHANGES_REQUESTED** [review]

File: Configuration/SecurityController.m (hunk 1 above)
Comment (kfa-onewelcome) [reply Configuration/SecurityController.m:11]: it's, the Configurator v5 did remove this

File: OneginiExampleAppSwift/Assembly/PresenterAssembly.swift (hunk 2 above)
Comment (kfa-onewelcome) [reply OneginiExampleAppSwift/Assembly/PresenterAssembly.swift:97]: nope :P

File: OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift (hunk 3 above)
Comment (kfa-onewelcome) [reply OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: that's just an example of usage, one can change it to whatever he wants if implemented in the destination app

File: widget/ResourceGateway.swift (hunk 4 above)
Comment (kfa-onewelcome) [reply widget/ResourceGateway.swift:31]: well, the compiler claims that the responseData has need to be unwrapped before use ?

File: OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift (hunk 3 above)
Comment (kfa-onewelcome) [reply OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: ...

File: widget/ResourceGateway.swift (hunk 4 above)
Comment (szymon-onewelcome) [reply widget/ResourceGateway.swift:31]: Oh, right, now I see, my bad.

File: OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift (hunk 3 above)
Comment (szymon-onewelcome) [reply OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: Hmm I wish we had SDK in Swift :)

Comment (szymon-onewelcome) [reply OneginiExampleAppSwift/Interactors/RegisterUserInteractor.swift:47]: That was a test :)

**Review by szymon-onewelcome — APPROVED** [review]

**Pull request merged**